Trappist/src/Promact.Trappist.DomainModel/ApplicationClasses/TestSettings/TestSettingsAC.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Promact.Trappist.DomainModel.Models.Test;
+
+namespace Promact.Trappist.DomainModel.ApplicationClasses.TestSettings
```

Comment:
I need an Application class which will be inheriting Test class for implementing Automapper functionality in order to update the values of all the fields of Test Settings at once in database.